src/main/resources/scripts/vanilla gui.sk

```diff
@@ -0,0 +1,15 @@
+#
+# A vanilla example of a gui creation using Skript only
+# Requires Skript 2.3+
+#
+
+command /gui:
+	trigger:
+		open chest inventory with 6 row with name "Gui Test" to player
+		set slot 0 of open inventory of player to stone
```

Comment:
After a change of browser and desktop mode, here I am:
Like I said, local variables for the inventory and metadata for checking it.

Reply:
Updated